less/navigation.less

```diff
@@ -1 +1,42 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+header{
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    border-bottom: 3px dashed @navigation-border;
+    @media @mobile {
+        width: @default-width;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        color: red;
+        h1{
+            font-size: 3rem;
+        }
+        nav{
+            width: 100%;
+            display: flex;
+        }
+    }
+    nav {
+        align-items: center;
+        width: 75%;
+        display: flex;
+        justify-content: space-between;
+        a{
+            text-decoration: none;
+            color: @shark;
+            font-size: 1.4rem;
+            &:hover{
+                .keyframes(fade-in;{
```

Comment:
love the animation and the fact you went above and beyond to make keyframes work with LESS